ext/standard/basic_functions.stub.php

```diff
@@ -60,6 +60,24 @@ function stream_wrapper_restore(string $protocol): bool {}
 /** @return int|false */
 function array_push(array &$stack, ...$args) {}
 
+function krsort(array &$stack, int $sort_flags = PHP_SORT_REGULAR): bool {}
```

Comment:
We can't typehint reference arguments. See https://github.com/php/php-src/pull/4501#discussion_r312705051.

Reply:
Well, krsort() expects an array *on* *entry*, so in this case the type declaration should be fine.

Reply:
but array_push function has typehint